examples/src/main/java/io/grpc/examples/experimental/CompressionServerInterceptor.java

```diff
@@ -0,0 +1,40 @@
+package io.grpc.examples.experimental;
+
+import io.grpc.Metadata;
+import io.grpc.ServerCall;
+import io.grpc.ServerCall.Listener;
+import io.grpc.ServerCallHandler;
+import io.grpc.ServerInterceptor;
+
+public class CompressionServerInterceptor implements ServerInterceptor {
+
+    private static final String GZIP = "gzip";
+
+    @Override
+    public <ReqT, RespT> Listener<ReqT> interceptCall(ServerCall<ReqT, RespT> call, Metadata headers,
+        ServerCallHandler<ReqT, RespT> next) {
+
+        if (isGzipAcceptable(headers)) {
+            call.setMessageCompression(true);
```

Comment:
This is unnecessary. This is the default.

Reply:
Removed the unnecessary configuration as the review comment